daemon/src/maker_inc_connections.rs

```diff
@@ -489,17 +489,16 @@ async fn upgrade(
         .context("Stream closed before first message")?;
 
     match first_message {
-        wire::TakerToMaker::Hello(taker_version) => {
-            let our_version = Version::current();
-            write
-                .send(wire::MakerToTaker::Hello(our_version.clone()))
-                .await?;
-
-            if our_version != taker_version {
-                bail!(
-                    "Network version mismatch, we are on version {our_version} but taker is on version {taker_version}",
-                );
-            }
+        wire::TakerToMaker::Hello(taker_wire_version) => {
+            tracing::info!(taker_id = %taker_id, %taker_wire_version, "Received Hello message from taker (version <=0.4.7");
```

Comment:
❓ Can't you replace `taker_id = %taker_id` with just `%taker_id`?

Reply:
changed in all 3 cases (added a separate commit)